content/millennium/r4/clinical/diagnostics/observation.md

```diff
@@ -340,5 +342,6 @@ The common [errors] and [OperationOutcomes] may be returned.
 [FHIR<sup>®</sup> Update]: https://hl7.org/fhir/R4/http.html#update
 [Extensions for converting between versions]: https://www.hl7.org/fhir/r4/versions.html#extensions
 [R5 Snapshot of Observation.value]: https://hl7.org/fhir/2020Feb/observation-definitions.html#Observation.value_x_
+[convertedMeasurement]: http://hl7.org/fhir/StructureDefinition/iso21090-PQ-translation
```

Comment:
the link should be https, same for performerFunction below

Reply:
Fixed for both here: c4ddf22cb8f36813c353e6e1510b96a5bb23e4bd